README.md

```diff
@@ -93,7 +123,16 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
+```
+Calling a function is executing a function, while referencing is simply stating the existence of a function.
+
+Calling:
+
+vegetables()
+
+referencing:
+
+function vegetables =
```

Comment:
this is defining a function. you would see referencing with an event listener, for example:

`document.body.addEventListener("click", vegetables)`